pkg/network/protocols/http/testutil/testutil.go

```diff
@@ -84,11 +101,17 @@ func HTTPServer(t *testing.T, addr string, options Options) func() {
 			return err
 		}
 	}
-	err := listenFn()
+	err = listenFn()
 	if err != nil {
 		t.Fatalf("server listen: %s", err)
 	}
-	return func() { srv.Shutdown(context.Background()) }
+	return func() {
+		srv.Shutdown(context.Background())
+		err := sysctl.Set("net.ipv4.tcp_timestamps", oldTCPTS)
+		if err != nil {
+			t.Fatal("can't set old value of TCP timestamp", err)
```

Comment:
no need to fail here, it's not related to the test, and it can cause flakiness.
instead an error **log** can suffice